src/yy/tidialogs/MultiPickerProxy.java

```diff
@@ -56,11 +53,11 @@ public void processProperties(KrollDict properties) {
 			String cancelButtonTitle;
 			String message;
 			boolean cancellable = true;
-			if (properties.containsKeyAndNotNull("onchange")) {
-				Object o = properties.get("onchange");
+			if (properties.containsKeyAndNotNull("onChange")) {
+				Object o = properties.get("onChange");
```

Comment:
If you are changing the event, why not change it to simply `change` to keep it line with the rest of the TiSDK?
Also it will need a more significant version bump if the api is changing.